### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by ousmane0126** [github/india#774] [GitHub Platform]
What's The Problem
![Screenshot_20220213-094444.png](https://user-images.githubusercontent.com/96368441/153745934-0332a714-002b-48a2-8679-3862021eceef.png)

**Comment by ey-mentor1** [comment]
Please check the Authorization of the account.
As per error, the password is not supported for this user.

**Comment by FRTMentor** [comment]
@ousmane0126 your credential was removed from the GitHub account or it was expired.
Please reset the account or create new one.
